bin/lhapdf.in

```diff
@@ -286,7 +286,7 @@ def download_url(source, dest_dir, dryrun=False):
         logging.debug('Downloading from %s' % url)
         logging.debug('Downloading to %s' % dest_filepath)
         if dryrun:
-            logging.info('%s [%s]' % (os.path.basename(url), convertBytes(file_size)))
+            #logging.info('%s [%s]' % (os.path.basename(url), convertBytes(file_size)))
```

Comment:
Does this still have to be commented out? The default value of -1 should make this work.